### DISCUSSION TITLE
Optimization issues.

### DISCUSSION THREAD

**Discussion by Aendar** [HolographicWings/TOTK-Mods-collection#136] [Q&A]
I can't even describe my problem intelligently, so I apologize in advance.
Well, I just can't get good FPS values, can you suggest what I'm doing wrong and how to fix this misunderstanding.
I've read all the recommendations, tried a ton of configurations, watched a bunch of videos, but still can't get at least stable 40-45 FPS.
Template from survey:
- CPU : AMD Ryzen 7 5800H
- GPU : RTX 3050 TI 4GB
- RAM : 16GB, 3200
- Emulator :  yuzu 3754
- Average framerate : 25-30 in wilds, 18-25 in towns 
- API : Vulkan

I saw how "hdmonteiro" on the same configuration achieved 50-60 fps https://github.com/HolographicWings/TOTK-Mods-collection/issues/72#issuecomment-1595351897, I don't know how to tag people on github, ?private messages are closed (I don't know if there are private messages on github)?, so I couldn't ask directly.
How to achieve stable frames, please give at least a vector of movement to find information on this topic.
All drivers are updated, the platform - Windows 11 (licensed), nothing hanging in the tray, no trojans, the laptop is always connected to the mains.

**Comment by Aendar** [comment]
I have forgotten to upload configurations screenshots. CPU accuracy on Auto.
![image](https://github.com/HolographicWings/TOTK-Mods-collection/assets/96650425/4568bfbc-de09-4aeb-ab80-09332456df73)
![image](https://github.com/HolographicWings/TOTK-Mods-collection/assets/96650425/85a36fcd-f93f-4404-a3c5-ca40f1f3712f)

**Comment by SpockBauru** [comment]
Sorry for the delay, it was a wild week and suddenly everything changed. Try the new recommended settings, maybe it helps: https://github.com/HolographicWings/TOTK-Mods-collection/blob/main/SETUP.md

**Reply by Aendar** [reply]
Thank you very much for the answer!
I was able to partially solve the problem, it was about throttling, after lowering CPU consumption and everything became fine.

**Comment by Fasty187** [comment]
For all people who have problems with lags, especially in the main menu, map or quick selection, I can recommend setting the page file to 32 GB. Solved all my problems!
